include/deal.II/grid/reference_cell.h

```diff
@@ -450,6 +450,22 @@ class ReferenceCell
    * @{
    */
 
+  /**
+   * Return true if the given point is inside the reference cell of the present
+   * space dimension. This function accepts an additional parameter (which
+   * defaults to zero) which
+   * specifies by how much the point position may actually be outside the true
+   * reference cell. This is useful because in practice we may often not be able
+   * to compute the coordinates of a point in reference coordinates exactly, but
+   * only up to numerical roundoff.
+   *
+   * The tolerance parameter may be less than zero, indicating that the point
+   * should be safely inside the cell.
+   */
+  template <int dim>
+  bool
+  point_is_inside(const Point<dim> &p, const double eps = 0) const;
```

Comment:
Maybe formulate as question `is_point_inside()`.

Reply:
I don't like `is_point_inside` as a name. It sounds grammatically wrong. 

Would `contains_point` be better? As in
```
  cell->contains_point(p)
```

Reply:
Maybe just is_inside(), in analogy to std::is_same, std::is_enum, etc.

(point.is_inside(cell) would be easier to read, but does not make sense from a software design perspective. I assume this is the reason why you repeated „point“ in the function name?)